.github/workflows/cache.yml

```diff
@@ -1,20 +0,0 @@
-name: Cache Keyman API
-
-on:
-  schedule:
-    - cron: '0 0 * * *'
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Fetch from Keyman API
-        run: |
-          curl https://api.keyman.com/search?q= > src/gen/cached-keyman-api.json
-
-      - uses: stefanzweifel/git-auto-commit-action@v4
-        with:
-          commit_message: Update Keyman API
```

Comment:
This file now lives here: https://github.com/eddieantonio/cache.predictivetext.studio/blob/f19efcf5b50fc90d55c2d9403ad2748046c3100d/.github/workflows/cache.yml